install.sh

```diff
@@ -208,9 +198,14 @@ echo crew >> .git/info/sparse-checkout
 echo tools   >> .git/info/sparse-checkout
 git fetch origin "${BRANCH}"
 git reset --hard origin/"${BRANCH}"
+echo -e "${YELLOW}Updating crew package information.${RESET}"
 crew update
 
-yes | crew install $LATE_PACKAGES 
+crew_installed_packages=$(jq  '.installed_packages[] .name' "${CREW_CONFIG_PATH}/device.json" | sed -e 's/"//g'   | sort)
+crew_core_packages=$(cat "${CREW_LIB_PATH}/tools/core_packages.txt" | sort)
+# The comm tool gives the set of packages in core not already installed.
+# Install those remaining core packages.
+yes | crew install $(comm -23 <(echo $crew_core_packages) <(echo $crew_installed_packages))
```

Comment:
Very interesting.  The only drawback I can see of doing it this way is if order matters during install.

Reply:
Crew handles the order anyways though... And we're not breaking crew, git, or curl any more with ordering...